manimlib/__init__.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
-import manimlib.config
-import manimlib.constants
-import manimlib.extract_scene
+from . import config
+from . import constants
 
 
 def main():
-    args = manimlib.config.parse_cli()
-    config = manimlib.config.get_configuration(args)
-    manimlib.constants.initialize_directories(config)
-    manimlib.extract_scene.main(config)
+    args = config.parse_cli()
+    cfg = config.get_configuration(args)
+    constants.initialize_directories(cfg)
+    from . import extract_scene
```

Comment:
There are a few factors at play which lead to this import being moved here:
* [The `from x import y` syntax creates a **copy** of y in the importing module's global scope](https://stackoverflow.com/a/3536638). This means that if y is imported and later updated the importing module's copy will be stale.
* The output directories aren't initialized from "" until `initialize_directories()` is called.
* `extract_scene` imports `scene.scene` imports `scene.scene_file_writer` imports `VIDEO_OUTPUT_DIR`.

So if `extract_scene` was imported at the top of the file scene_file_writer would have "" for `VIDEO_OUTPUT_DIR`, which will crash.

The constants file is the only place I know of where this is a problem. It'd probably be best to either refactor extract_scene.py so that it doesn't import scene.scene or find a better way to pass the output directories, for example in the config for Scene.

Reply:
I guess that would be good, in due time anyway.